### ISSUE TITLE
Autoscrolling doesn't seen to work when dragging an item to the bottom of the list

### ISSUE THREAD

**Issue by scotttball** [computerjazz/react-native-draggable-flatlist#140]
**Describe the bug**
When dragging an item to the edge of the bottom screen, it doesn't seen to autoscroll. I don't know if it's an issue with the ref or which figuring out the parent component ref. 

**To Reproduce**
I don't have a snack, but I'm figuring it's just a misconfiguration with how I'm implementing it. I'll dig through your code, but I have to admit, I'm not an expert at reanimated and most of the variables are reanimated values determining if it's the edge of the screen.

**Platform & Dependencies**
Please list any  applicable dependencies in addition to those below (react-navigation etc).
 - Platform: IOS & Android
 - React Native or Expo version: ^0.59.9
 - Reanimated version: ^1.7.0"
 - React Native Gesture Handler version: ^1.5.6

**Additional context**

Basically dragging an item to the end of the screen doesn't autoscroll to the next set of items. I can upload a generic video, but I prefer to just email you it (for privacy reasons).

Let me know.

Again, Thanks for your work.

Scott

**Comment by scotttball** [comment]
I'm closing this. I tested this on a snack and it seems like the activation distance to autoscroll is just very small for what I'm using it for. It works though.
